src/vllm_haystack/__init__.py

```diff
@@ -34,6 +34,7 @@ class vLLMInvocationLayer(PromptModelInvocationLayer):
     def __init__(
         self,
         api_base: str,
+        api_key: str,
```

Comment:
Could we make the `api_key` optional as an parameter, e.g. `Optional[str]` and default it to `None`?

Reply:
Updated! Thanks for the quick feedback @LLukas22!